content/en/blog/2025/contribex-survey-results/index.md

```diff
@@ -0,0 +1,288 @@
+---
+
+The OpenTelemetry Contributor Experience SIG recently surveyed the community to
+learn what it's like to contribute to the project and what we can do to improve
+the contributor experience. The Contributor Experience Survey asked contributors
+for their thoughts about the project's organization, SIG contributions,
+leadership committee interactions, and event attendance. We received 120
+responses and heard from all 47 Special Interest Groups (SIGs). We'll use this
+feedback to make contributing to OpenTelemetry easier and more rewarding. A big
+thank you to everyone who participated in the survey! Let's review the results.
+
+## Key takeaways
+
+We conducted this survey to learn what the project can do better. Here are the
+key things we learned, along with areas we can improve for contributors:
+
+- **Getting started**: 48% of newer contributors don't confidently know how to
+  get started. Intra- and inter-SIG communication and contributing guides need
+  improvement.
+- **SIG meetings**: 82% of contributors find SIG meetings useful. Meeting
+  attendance correlates to greater knowledge about navigating and contributing
+  to the project, but some dislike the synchronicity meetings require.
+- **Workflows**: More than half of contributors do not receive timely feedback
+  on their contributions. They also experience project tooling headaches and
+  gatekeeping bottlenecks.
+- **Project and subproject priorities**: Less than half of maintainers can
+  confidently say they know the priorities of OpenTelemetry's leadership
+  committees. Contributors also say that it's difficult to find subproject
+  priorities.
+- **Events**: Contributors want more in-person events that are focused on
+  collaboration rather than talks.
+
+## Detailed insights
+
+With a total of 42 questions, this survey covered the breadth of the
+OpenTelemetry contributor experience. In this section, we analyze the responses
+and summarize what we learned.
+
+In some categories, the sample size is quite small and therefore the margin of
+error is high. We have included the number of responses in parentheses wherever
+possible.
+
+For our analysis, we treated self-reported data as true and combined “agree”
+with “strongly agree” responses, as well as “disagree” with “strongly disagree.”
+So when we say, for example, “contributors who attend SIG meetings are far more
+likely to know how to begin contributing in any SIG,” what we mean is that
+contributors who _agreed or strongly agreed_ that they regularly attend SIG
+meetings are more likely to _agree or strongly agree_ that they know how to get
+started contributing.
+
+### About the respondents
+
+We heard from 120 contributors, including 45 maintainers and at least one
+contributor from each SIG. Here's more about our respondents:
+
+- 63% work for an observability vendor.
+- 20% have been contributing for less than six months.
+- 86% live in North America (45%) or EMEA (41%). These numbers roughly match our
```

Comment:
@tiffany76 I got a little confused when I first read this sentence by the percentages in brackets. After second reading it was all clear though. I am wondering if there is a way to make it easier to read. Perhaps something like this:

```suggestion
- 86% live in North America or EMEA (45% and 41% respectively). These numbers roughly match our
```